### ISSUE TITLE
Review button on Contract interaction screen is enabled without contract address

### ISSUE THREAD

**Issue by liliya-soroka** [gnosis/safe-react-apps#144]
## Description
Review button on Contract interaction screen is enabled without contract address. The button is enabled but nothing happens on Review button click
## Environment
 - Browser: Chrome
 - Wallet: MetaMask
 - Safe: 0x1230B3d59858296A31053C1b8562Ecf89A2f888b
 - Environment:
   - production (rinkeby)

## Steps to reproduce
 1. Go to the contract interaction
 2.  past ABI
 3. select method
 4. enter valid parameters
 Current result: Review button is enabled without contract address ( the field is empty as ABI was used)

5. Click Review
Current result: Nothing happens as the contract address is required

## Expected result
Review button should be enabled only if the all mandatory, valid information are entered

## Screenshots
![image](https://user-images.githubusercontent.com/338622/124154628-f4faee00-da95-11eb-879c-8e927a69e805.png)

**Comment by mikheevm** [comment]
Should this issue be a part of safe-react and not apps?

**Comment by yagopv** [comment]
Yeah that's not part of the apps repos right @dasanra? ... should be moved ?
